NeoML/src/Dnn/DnnSolver.cpp

```diff
@@ -313,6 +315,8 @@ void CDnnSolver::Serialize( CArchive& archive, CDnn& dnn )
 
 		layerToParamDiffBlobsSum.DeleteAll();
 		layerToGradientHistory.DeleteAll();
+		layersToReduce.DeleteAll();
+		reduceOrder.DeleteAll();
```

Comment:
I usually use `Empty()`. Here I used `DeleteAll()` because of 2 lines above. Why 2 methods do the same thing? I don't know, it is some legacy old interface.

Reply:
Do you have a limit for keyboard clicks?. You may change both above lines to `Empty()`.